task-09/README.md

```diff
@@ -0,0 +1,57 @@
+[previous task](../task-08)
+
+# Task 9: RIOT and RPL
+
+This task will demonstrate basic routing functionalities in RIOT provided by the [RPL](https://tools.ietf.org/html/rfc6550) (IPv6 Routing Protocol for Low-Power and Lossy Networks) routing protocol and multihop communication.
+
+It uses an adapted version of the default [`gnrc_networking`](https://github.com/RIOT-OS/RIOT/tree/master/examples/gnrc_networking) example in the RIOT repository.
+
+## Prerequisites
+
+* Compile the application and run on your `BOARD`
+* Build groups of at least three persons
+* Each group should change to an individual RF channel. It can be changed by e.g.
+
+```
+ifconfig 7 set chan 11
+
+```
+
+* Check if this worked by looking at the output of the `ifconfig` command. Also have a look at the preconfigured IPv6 addresses that all have a local scope.
+
+
+## Task 9.1: Initialize RPL
+
+* In order to use RPL we have to choose **one** RPL root node and configure a global IPv6 address for it. This is simply done by adding the address to the network interface (`7`) via `ifconfig`:
+(Please note that the network interface is `6` when running on native)
+
+```
+ifconfig 7 add 2001:db8::1
+```
+
+* Next we need to initialize RPL on the network interface (`7`) for **all** nodes: 
+
+```
+rpl init 7
+```
+
+* To start the routing process and the generation of a RPL DODAG (Destination Oriented Directed Acyclic Graph), we need to start the root node with an Instance ID (here `1`) and a DODAG ID (here its global IPv6 address) 
+
+```
+rpl root 1 2001:db8::1
+```
+
+## Analyse the topology
+
+* On not-root nodes, look at the output of `ifconfig` again. You should see that a global unicast IPv6 address has been set automatically.
+
+* Typing `rpl` gives you some information about the nodes position in the DODAG. Check the local IPv6 address of the selected parent node. Look for value `R:` which describes the "rank" or more general, the hop count metric. What is the increment for one hop?
```

Comment:
Look for value `R:` which describes the "rank" or more general, ~~the hop count metric.~~ the position in the DODAG according to the metric.